Tests/LibWeb/Text/input/GamepadAPI/gamepad-iframe.html

```diff
@@ -30,14 +30,22 @@
 
```

Comment:
Commit title: nothing in it suggests a test change; maybe mark it as such?

E.g.
```
Tests/LibWeb: Prevent flake with iframe loading in gamepad-iframe.html
```

+ optionally an explanation of what was going on in the commit message.